integration/docker/cgroups_test.go

```diff
@@ -204,3 +205,32 @@ var _ = Describe("Checking CPU cgroups in the host", func() {
 		})
 	})
 })
+
+var _ = Describe("Check cgroup paths", func() {
+	var (
+		args []string
+		id   string
+	)
+
+	BeforeEach(func() {
+		id = randomDockerName()
+		args = []string{"-d", "--name", id}
+	})
+
+	AfterEach(func() {
+		Expect(RemoveDockerContainer(id)).To(BeTrue())
+		Expect(ExistDockerContainer(id)).NotTo(BeTrue())
+	})
+
+	DescribeTable("with a parent cgroup",
+		func(parentCgroup string) {
+			Skip("Issue: https://github.com/kata-containers/runtime/issues/1365")
+			args = append(args, "--cgroup-parent", parentCgroup, Image)
+			_, _, exitCode := dockerRun(args...)
+			Expect(exitCode).To(BeZero())
+		},
+		withParentCgroup("../../../../"),
```

Comment:
Does it have to be "four directories up"? What about `"../"`, `"../../"`, *etc*?